File: 0609/함수.c
#include<stdio.h>

int add(int a, int b); // 함수 선언 (함수 프로토타입)
int main(void){
  int result = add(1, 3); // 메인함수
  printf("%d\n", result);
  return 0;
}

int add(int a, int b) // 함수 정의
{
  return a+b;
}

File: 0618/프로그래밍B/함수.py
# 두 수의 합계를 구하는 프로그램

def sum(a,b):
  return a+b
print(sum(10,5))

# 세수를 입력받고 평균을 구하는 프로그램

def avg(m, s, i):
  average = (m+s+i)/3
  return average
mat = int(input("수학 점수 입력: "))
sci = int(input("과학 점수 입력: "))
info = int(input("정보 점수 입력: "))
result = avg(mat, sci, info)
print(result)


